app/views/shared/_sidebar_navigation.html.erb

```diff
@@ -1,5 +1,7 @@
 <% content_item = @content_item.content_item.parsed_content %>
 
 <div class="govuk-grid-column-one-third">
+  <%= yield :govuk_chat_promo %>
```

Comment:
Is the banner consistently here in also usages of this app? (I have some recollection of other placings but they may all be in collections)